articles/inter-vlan-routing/index.md

```diff
@@ -0,0 +1,160 @@
+date: 2021-03-16T00:00:00-10:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/inter-vlan-routing/hero.jpg
+    alt: Network Connection example image
+---
+As a network administrator, one of the tasks is to segment a single switched Local Area Network (LAN) logically without having to run new cables and changing the existing network. The concept of "VLAN" and its configurations will help one handle the task at hand with ease.
+<!--more-->
+In this article, we will learn about what inter-VLAN routing is, various methods for VLAN routing, and also build a network configuration to understand how it works.
+
+# Table of contents
+- [Prerequisite](#prerequisite)
+- [What is inter-VLAN routing?](#what-is-inter-vlan-routing)
+- [Methods of inter-VLAN routing](#inter-vlan-routing-methods)
+- [Inter-VLAN network configurations](#network-configurations-for-inter-vlan-communication-using-router-on-stick-method)
+- [Conclusion](#conclusion)
+
+### Prerequisite
+As a prerequisite, the reader must have a good understanding of basic networking concepts, and how to work with Cisco packet tracer.
+
+To download Cisco packet tracer visit [this link](https://www.computernetworkingnotes.com/ccna-study-guide/download-packet-tracer-for-windows-and-linux.html) to install the latest version installed on your computer.
+
+### What is Inter-VLAN routing?
+Virtual Local Area Networks (VLANs) are network segments on a switched LAN that groups different hosts together logically, regardless of their physical locations on the network.
+
+VLANs make it easier to segment a workstation into different isolated LANs. They improve the performance of the network and are flexible, since they are logically connected.
+
+VLANs act as separate subnets on the network. To move packets from one VLAN to another and enable communications between hosts, we have to configure the network to enable inter-VLAN routing.
+
+### Inter-VLAN routing methods
+#### Legacy inter-VLAN routing
+In this method, multiple router interfaces are used, each connecting to a switch port in different VLANs. These interfaces are served as default gateways. Sometimes, it may lead to network issues, since expanding the network requires additional cabling.
+
+#### Router-on-a-stick
+In this method, unlike the legacy routing, one physical interface port is used for routing the traffic between the network segments. The network administrator need not create separate VLAN interfaces like `fa0/1` to `fa0/10`. Instead, all the interfaces from 1 to 10 are created with a single interface.
+
+#### Layer 3 switch using Switched Virtual Interface (SVI)
+Currently, this method of inter-VLAN routing that uses layer 3/multilayer switch and Switched Virtual Interfaces (SVI) are being used more commonly. SVIs are created for VLANs that exist on the switch and they perform the same function for the VLANs as that of a router.
+
+Layer 3 switches are expensive, hence not affordable.
```

Comment:
If you are mentioning about affordability. Make sure you speak about them in the previous 2 methods too. So, that the reader will be able to compare different methods.